### DISCUSSION TITLE
Is the project still active/maintained?

### DISCUSSION THREAD

**Discussion by l3ender** [reposaur/reposaur#86] [General]
Hello! We are wanting to start ensuring compliance on our GitHub repositories. Reposaur looks like a great tool, but we aren't sure if the project is still active. In looking at the open issues and pull requests, it appears there may not be any active maintenance on the project.

While we realize maintaining software is time consuming and often thankless, we are hesitant to start using the tool if it is no longer maintained. So we wanted to see if there is any activity or maintainer which could share input.

Thank you!

**Comment by crqrdotcom** [comment]
Hey @l3ender - while the project hasn't had any activity for quite awhile, we've been drafting a new version, with well-defined boundaries, extensibility and stability in mind.

We'll be sharing more details publicly in the repository discussions in the next couple of weeks. 

The project is backed by @glisshq - a portuguese consulting company.

Feel free to reach out if you want to know more, share your use cases or feedback in general: joao@gliss.io.

Thanks for your interest 🎉

**Comment by l3ender** [comment]
Thanks for the update, and looking forward to seeing what evolves!
